src/cloud-api-adaptor/install/yamls/caa-pod.yaml

```diff
@@ -79,3 +83,9 @@ spec:
       - hostPath:
           path: /run/netns
         name: netns
+      # # setting for cloud provider runtime plugin
```

Comment:
Please change the runtime plugin to external plugin. And the compile time plugin to built-in plugin across the code

Reply:
updated.